parts/k8s/cloud-init/artifacts/sshd_config

```diff
@@ -72,6 +91,5 @@ Subsystem sftp /usr/lib/openssh/sftp-server
 # PAM authentication, then enable this but set PasswordAuthentication
 # and ChallengeResponseAuthentication to 'no'.
 UsePAM yes
-
-# CLOUD_IMG: This file was created/modified by the Cloud Image build process
```

Comment:
I see we moved `ClientAliveInterval` up above. I assume it's ok to remove this comment?